### ISSUE TITLE
Errors with get meaning "become"

### ISSUE THREAD

**Issue by amir-zeldes** [UniversalDependencies/UD_English-EWT#105]
I was just looking for some example constructions for "I got" and found some errors:

  * I got confused - confused is obj, should be xcomp
  * I got lucky - lucky is obj, should be xcomp
  * I got so excited - got is aux to excited, which is tagged as ADJ. Got should be the head IMO, and have excited as xcomp, or if excited is seen as a passive, then excited should be tagged VBN and deprels should be aux:pass + nsubj:pass

**Comment by nschneid** [comment]
See also UniversalDependencies/docs#647

**Comment by nschneid** [comment]
For "I got so excited" I would not consider "got" to be passive—more like "became".

**Comment by manning** [comment]
I agree. All of these should be xcomp! But we should also have aux for passive auxiliary cases.

**Comment by amir-zeldes** [comment]
Definitely, and we distinguish them in GUM as well:

  * xcomp (incl. causative cases): https://corpling.uis.georgetown.edu/annis/#_q=bGVtbWE9ImdldCIgLT5kZXAgdG9rX2Z1bmM9Inhjb21wIg&_c=R1VN
  * passive: https://corpling.uis.georgetown.edu/annis/#_q=bGVtbWE9ImdldCIgXz1fIHRva19mdW5jPSJhdXg6cGFzcyI&_c=R1VN

I should note we essentially co-determine the xpos for the predicate when we use `aux:pass` (the head is then VBN), but there are edge cases. For example we have some cases of 'get dressed' which are analyzed as passive mainly because 'dressed' is canonically tagged as VBN in PTB, so we stuck to that in GUM too, even though it's not really possible to add the by-agent IMO.

**Comment by nschneid** [comment]
(I checked and the 3 errors in the first post are fixed)
